hack/jenkins/e2e.sh

```diff
@@ -1177,45 +1178,6 @@ if [[ "${E2E_UP,,}" == "true" || "${JENKINS_FORCE_GET_TARS:-}" =~ ^[yY]$ ]]; the
             exit 1
         fi
 
-        # Tell kube-up.sh to skip the update, it doesn't lock. An internal
-        # gcloud bug can cause racing component updates to stomp on each
-        # other.
-        export KUBE_SKIP_UPDATE=y
```

Comment:
This one you should keep, at least until we remove the `gcloud components install` calls from [cluster/gce/util.sh](https://github.com/kubernetes/kubernetes/blob/master/cluster/gce/util.sh#L82)